setup.py

```diff
@@ -30,19 +30,21 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
+
 setup(name="ducktape",
       version=version,
       description="Distributed system test tools",
       author="Confluent",
-      platforms=["any"], 
+      platforms=["any"],
       entry_points={
-        'console_scripts': ['ducktape=ducktape.command_line.main:main'],
+          'console_scripts': ['ducktape=ducktape.command_line.main:main'],
       },
       license="apache2.0",
       url="http://github.com/confluentinc/ducktape",
       packages=find_packages(),
       package_data={'ducktape': ['templates/report/*']},
       install_requires=['jinja2', 'requests', 'paramiko', 'pysistence', 'pyzmq'],
-      tests_require=['pytest', 'mock', 'psutil==4.1.0', 'memory_profiler==0.41', 'statistics', 'requests-testadapter', 'boto3', 'pycrypto', 'pywinrm'],
+      tests_require=['pytest', 'mock', 'psutil==4.1.0', 'memory_profiler==0.41',
```

Comment:
If you add `flake8` to `setup_requires`, per https://flake8.readthedocs.io/en/2.0/setuptools.html, you can get a `setup.py` command for doing flake8. This is also nice since in the Jenkins jobs you don't need to separately install flake8.